src/pickup_meta.rs

```diff
@@ -260,6 +260,18 @@ impl PickupType
             PickupType::ShinyMissile,
         ].iter().map(|i| *i)
     }
+
+    pub fn from_string(string: String) -> Self {
+        for i in PickupType::iter() {
+            if i.name().to_string().to_lowercase() == string.to_lowercase() {
+                return i;
+            }
+        }
+
+        println!("Unknown Item Type - {}", string);
+        assert!(false);
+        PickupType::Nothing
```

Comment:
This should be an explicit panic instead of an `assert!(false)`. Alternatively, maybe this function should return `Result<Self, String>` instead.